tutorial.md

```diff
@@ -0,0 +1,191 @@
+# Hello Cloud Demo Step-by-Step Tutorial
+
+This is a step-by-step tutorial on how to build and deploy this project to the cloud
+
+---
+
+# <a name="prereq"></a> Pre-Requisites
+
+- Docker
```

Comment:
Maybe we can organize the pre-reqs into CLI tools, accounts, and services to provision?

CLI tools would be docker, java, git,curl, maven, helm, IBM cloud CLI, kubectl
Accounts: DockerHub, IBM Cloud
Services to provision: Cloudant database, Kubernetes Cluster

Since we give instructions on how to install IBM Cloud CLI, kubectl and create a kube cluster, maybe we can move those into a separate section (not a pre-req)? My understanding is that pre-reqs are things we expect the reader to already have ready and we won't be giving instructions. Before Step 0-6, we can mention that this section goes over how to login to IBM Cloud CLI, create a kube cluster, etc?

Reply:
When it comes to the presentation we still consider them as "pre-req". I re-worded/ordered to be a little bit clearer on what they mean

Reply:
lgtm!